src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+
+    public Lottery getRandomBall() {
+
+        Random random = new Random();
+        int numberOfBall = random.nextInt(100);
+
+        Lottery ball = new Lottery();
+        ball.setNumber(numberOfBall);
+        ball.setColor(ball.getRandomColor());

```

Comment:
ball.getRandomColor() is not a valid method. This method should belong to ColorSupplier